dotCMS/src/main/java/com/dotcms/rendering/velocity/servlet/VelocityServlet.java

```diff
@@ -61,6 +62,7 @@ public VelocityServlet() {
     public static PageMode processPageMode (final User user, final HttpServletRequest request) {
 
         final LoginMode loginMode = LoginMode.get(request);
+        Logger.debug(VelocityServlet.class, "VelocityServlet_processPageMode LoginMode: " + loginMode.toString());
```

Comment:
I would always use the supplier implementation 
 Logger.debug(VelocityServlet.class, ()->"VelocityServlet_processPageMode LoginMode: " + loginMode.toString());